samples/bot-teams-catering/csharp/Catering/Resources/Confirmation.json

```diff
@@ -0,0 +1,35 @@
+{
+  "type": "AdaptiveCard",
+  "appId": "86ee312e-e8f2-4a03-9515-4ac80986abd5",
+  "body": [
+    {
+      "type": "TextBlock",
+      "text": "Order confirmed",
+      "horizontalAlignment": "Center",
+      "size": "Large",
+      "weight": "Bolder"
+    },
+    {
+      "type": "TextBlock",
+      "text": "Thanks! You're all set.",
+      "horizontalAlignment": "Center"
+    },
+    {
+      "type": "ActionSet",
+      "actions": [
+        {
+          "type": "Action.Execute",
+          "title": "See Recent Orders",
+          "verb":  "order",
+          "data": {
+            "nextCardToSend": 3,
+            "currentCard": 4,
+            "option": ""
+          }
+        }
+      ]
+    }
+  ],
+  "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+  "version": "1.2"
```

Comment:
[Sequential workflow ](https://docs.microsoft.com/en-us/microsoftteams/platform/task-modules-and-cards/cards/universal-actions-for-adaptive-cards/sequential-workflows#sequential-workflow-for-adaptive-cards) is added as part of Adaptive Card schema v1.4. Could you please update all the adaptive card versions to 1.4?

Reply:
updated for all json